.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "go.inferGopath": false
```

Comment:
Was this file intentionally checked in?

Reply:
Removed the file and added it to `.gitignore`